_search-plugins/keyword-search.md

```diff
@@ -118,8 +118,13 @@ The following table lists the supported similarity algorithms.
 
 Algorithm | Description
 `BM25` | The default OpenSearch [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25) similarity algorithm. 
+`LegacyBM25` | (Deprecated) The older [LegacyBM25Similarity](https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/lucene/similarity/LegacyBM25Similarity.java) implementation. Kept for backward compatibility.
 `boolean` | Assigns terms a score equal to their boost value. Use `boolean` similarity when you want the document scores to be based on the binary value of whether the terms match.
 
+### Note
+
+OpenSearch 3.0 and later defaults to Lucene’s `BM25Similarity`. If required, you can opt in to `LegacyBM25` explicitly.
```

Comment:
I would highlight the difference.

Lucene recognized that scores were being multiplied by a constant factor and dropped it.

In OpenSearch 3.0, by default, all scores will be lower by a factor of 2.2 (if I remember correctly). It doesn't affect the order of results, unless you do something that depends on the specific values.

Examples include the `min_score` query that discards everything below a given score. It may also have implications for hybrid scoring (though I believe that's usually relative to the max score, so the relative scaling should be consistent).

Reply:
Updated @msfroh. Thanks